packages/core/src/components/overlay/overlay.tsx

```diff
@@ -295,26 +295,29 @@ export class Overlay extends React.Component<IOverlayProps, IOverlayState> {
         }
     }
 
-    private bringFocusInsideOverlay = () => {
-        const { containerElement } = this;
-
-        // container ref may be undefined between component mounting and Portal rendering
-        // activeElement may be undefined in some rare cases in IE
-        if (containerElement == null || document.activeElement == null || !this.props.isOpen) {
-            return;
-        }
+    private bringFocusInsideOverlay() {
+        // always delay focus manipulation to just before repaint to prevent scroll jumping
+        return requestAnimationFrame(() => {
```

Comment:
this fixed that nasty scroll jumping bug. simply delaying the actual `focus()` until the DOM has stabilized resolves everything. `rAF` to the rescue, again!